examples/joy-ui-nextjs/src/components/ThemeRegistry/theme.ts

```diff
@@ -0,0 +1,45 @@
+import { extendTheme } from '@mui/joy/styles';
+import { Inter, Source_Code_Pro } from 'next/font/google';
+
+const inter = Inter({
+  subsets: ['latin'],
+  adjustFontFallback: false, // prevent automatically adding Arial as the fallback
+  fallback: ['var(--joy-fontFamily-fallback)'], // explicitly stated we wish to use Joy UI's fallback
```

Comment:
I was suspicious about this, but it seems to make sense https://nextjs.org/docs/pages/api-reference/components/font#fallback

Reply:
But if `fallback = []`, then the resultant font family will have no fallback.

Reply:
What's the use case for fallback font?

Reply:
My understanding is that it generates the `font-family` value in this way:

```
Inter_<hash>, adjustFontFallback (Arial/Time New Roman), ...fallback
```

Reply:
From my understanding, the fallback will be used when the Inter is loading. What @mwskwong has done looks correct in my opinion.